.github/ISSUE_TEMPLATE/release-checklist.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Release checklist
+about: Prepare for a new release version of scpcaTools
+title: Prepare for scpcaTools release vX.X.X
+labels: release
+assignees: ''
+
+---
+
+## Steps for a new release of `scpcaTools`
+
+- [ ] Are all dependencies/issues expected for this release are resolved? Add any that are not resolved as dependencies for this issue.
```

Comment:
Do you mean mark this issue (the release) as blocked by the unresolved issues in ZenHub?

Reply:
Yes, I will reword.